RestCommClient/Classes/RCDevice.h

```diff
@@ -235,14 +235,22 @@ extern NSString* const RCDeviceCapabilityClientNameKey;
 /**
  *  Register the application for the push notifications
  * <b>firendly-name</b>: name of the client application
- * <b>username"</b>: username, for example: johndoe@telestax.com
```

Comment:
Did you remove username by mistake? Can you bring it back?

Reply:
No, we are getting that from RCDevice's @"signaling-username"; With this we sure use the same username which is logged in into app. Check lines 708, 709 in RCDevice.m

Reply:
Ah ok I see, cool

Reply:
Cool